ios/Sources/GutenbergKit/Sources/EditorLogging.swift

```diff
@@ -1,26 +1,73 @@
 import Foundation
+import OSLog
 
 /// Protocol for logging editor-related messages.
 public protocol EditorLogging: Sendable {
     /// Logs a message at the specified level.
     func log(_ level: EditorLogLevel, _ message: String)
 }
 
+extension Logger {
+
+    public static let performance = OSSignposter(subsystem: "GutenbergKit", category: "timing")
```

Comment:
Should this be a "performance" category? Or should it be removed entirely as a duplicate of `timing`? 
```suggestion
    public static let performance = OSSignposter(subsystem: "GutenbergKit", category: "performance")
```

Reply:
One is for logs, the other is for use with Instruments.app to add timing spans. They both fall under timing/performance, so while I don't feel strongly about what it's called I do think they should have the same category name (because they're not interchangeable).

Reply:
Ah, OK. I primarily thought it was a typo. I do not hold an opinion on the name.